ProcessLib/TES/TESProcess.cpp

```diff
@@ -322,7 +322,7 @@ GlobalVector const& TESProcess::computeVapourPartialPressure(
     std::vector<NumLib::LocalToGlobalIndexMap const*> const& dof_table,
     std::unique_ptr<GlobalVector>& result_cache)
 {
-    assert(&dof_table == _local_to_global_index_map.get());
+    assert(dof_table[0] == _local_to_global_index_map.get());
```

Comment:
I would prefer somewhere above: ```constexpr int process_id = 0;```
and then accessing the `dof_table` via `process_id` in the entire initializeConcreteProcess.

Reply:
:heavy_check_mark: